schema/release-schema.json

```diff
@@ -122,6 +122,14 @@
       "description": "Information from the planning phase of the contracting process. Note that many other fields can be filled in a planning release, in the appropriate fields in other schema sections; these would likely be estimates at this stage, e.g. value in tender.",
       "type": "object",
       "properties": {
+        "id": {
+          "title": "Planning ID",
+          "description": "An identifier for this planning process. This may be the same as the ocid, or may be an internal identifier for this planning.",
```

Comment:
I'm not sure the second sentence is necessary. I assume it's copied from `tender.id`, where it is necessary because the field is required, so publishers need guidance on how to populate the field even if the data source doesn't include it.

Reply:
Yes, the description comes from `tender.id`, and yes, the field is not mandatory here, but, I thought that it can still be the same as the `ocid`, as the planning would be the first stage of the contracting process, so the planning id is likely to be the ocid as well, but agree, it is not necessary to mention it.